testplan/web_ui/testing/src/Report/BatchReport.js

```diff
@@ -142,6 +142,11 @@ class BatchReport extends React.Component {
 
   render() {
     const {reportStatus, reportFetchMessage} = GetReportState(this.state);
+
+    if (this.state.report && this.state.report.name) {
+      window.document.title = this.state.report.name;
```

Comment:
just a question but is `window.document.title` any different from just `document.title`? https://developer.mozilla.org/en-US/docs/Web/API/Document/title

Reply:
> just a question but is `window.document.title` any different from just `document.title`? https://developer.mozilla.org/en-US/docs/Web/API/Document/title

Same thing, document === window.document
https://stackoverflow.com/questions/9895202/what-is-the-difference-between-window-screen-and-document-in-javascript